src/extra/widgets/msgbox/lv_msgbox.c

```diff
@@ -87,10 +103,15 @@ lv_obj_t * lv_msgbox_create(lv_obj_t * parent, const char * title, const char *
         lv_obj_align(label, LV_ALIGN_CENTER, 0, 0);
     }
 
-    mbox->text = lv_label_create(obj);
-    lv_label_set_text(mbox->text, txt);
-    lv_label_set_long_mode(mbox->text, LV_LABEL_LONG_WRAP);
-    lv_obj_set_width(mbox->text, lv_pct(100));
+    mbox->content = lv_obj_class_create_obj(&lv_msgbox_content_class, obj);
+
+    bool has_txt = txt && strlen(txt) > 0;
+    if (has_txt) {
+        mbox->text = lv_label_create(mbox->content);
+        lv_label_set_text(mbox->text, txt);
+        lv_label_set_long_mode(mbox->text, LV_LABEL_LONG_WRAP);
+        lv_obj_set_width(mbox->text, lv_pct(100));
+    }
 
     if(btn_txts) {
         mbox->btns = lv_btnmatrix_create(obj);
```

Comment:
Shouldn't it be created on `content` too?

Reply:
Text is now placed within `mbox->content`. What do you think?

Reply:
I mean the btnmatrix.

Reply:
@kisvegabor IMO it's better to put buttons outside. If the user want to, they can add complex layout inside `content` while still having consistent buttons. Also this is how Android design its dialog.

Reply:
Makes sense.